app/src/main/java/de/westnordost/streetcomplete/map/MapFragment.kt

```diff
@@ -350,6 +350,14 @@ open class MapFragment : Fragment(),
 
     /* ------------------------------- Controlling the map -------------------------------------- */
 
+    public fun adjustToOffsets(oldOffset: RectF, newOffset: RectF) {
+        controller?.screenCenterToLatLon(oldOffset)?.let { pos ->
+            controller?.updateCameraPosition {
+                position = controller?.getLatLonThatCentersLatLon(pos, newOffset)
+            }
+        }
```

Comment:
I changed these to call the controller methods directly instead of the MapFragment wrappers:

- `getViewPosition` ⇒ `screenCenterToLatLon` — I am confident this was the right choice, since it was the only use of that wrapper left.
- `getPositionThatCentersPosition` ⇒ `getLatLonThatCentersLatLon` — I think this was the right choice. The wrapper is only used in one other place (in `MainFragment`), so it opens the possibility of refactoring to remove it later.
- `updateCameraPosition` ⇒ `updateCameraPosition` — I am not sure this is a good choice. It's used in many places, so it's not going away any time soon. If it will always stay as a simple call down to the equivalent controller method, it doesn't matter either way, but if we might want to add additional behavior there, it would be better to call through the wrapper.
    - Counterpoint: I notice many of the `mapFragment` public methods are like this — they just call directly down to the equivalent controller method. I wonder if it makes sense to just expose a getter for the controller directly.

Reply:
> Counterpoint: I notice many of the mapFragment public methods are like this — they just call directly down to the equivalent controller method. I wonder if it makes sense to just expose a getter for the controller directly.

I don't think it makes sense to expose the controller to the outside. (Generally it seldomly does in my opinion)

Reply:
I don't mind, it's very short anyway